scripts/release/module-automation/commons.js

```diff
@@ -57,7 +58,8 @@ async function getPackageInfo(path) {
             url: repository?.url,
             testProjectUrl: testProject?.githubUrl,
             testProjectBranchName: testProject?.branchName,
-            changelogPath: `${path}/CHANGELOG.md`
+            changelogPath: `${path}/CHANGELOG.md`,
+            ...customData
```

Comment:
From what I can see, the only purpose to `customData` is including it in the return object. It feels unnecessary to make that a function parameter and not just include it after calling `getPackageInfo` manually. What's the reason for this approach?

Reply:
Yes I agree, updated accordingly 👌🏼

Reply:
And to answer your other question (Can't add a reply): 
Yes it's only adding support for Nanoflow Commons, and moving code to the shared file.
I did include the fix for your [other review ](https://github.com/mendix/widgets-resources/pull/953#discussion_r718267691) in this PR too, which is about removing the widgets' name from the widgets' changelog.

Reply:
Aight, thanks! Looks good 👍